scss/sections/_hero.scss

```diff
@@ -0,0 +1,54 @@
+// @import "/scss/_variables.scss";
```

Comment:
[REQ]: Remove comment -> good and clean good is going to self commented
[TIP]: If you don't know if code is good and has good naming that you should read it and your feeling should be like you has wrote good story :)